src/current/v23.2/show-virtual-cluster.md

```diff
@@ -0,0 +1,127 @@
+---
+title: SHOW VIRTUAL CLUSTER
+summary: The SHOW VIRTUAL CLUSTER statement provides details on virtual clusters and physical replication streams.
+toc: true
+docs_area: reference.sql
+---
+
+{{site.data.alerts.callout_info}}
+{% include feature-phases/preview.md %}
+{{site.data.alerts.end}}
+
+{% include_cached new-in.html version="v23.2" %} The `SHOW VIRTUAL CLUSTER` statement lists the virtual clusters running in a CockroachDB cluster. `SHOW VIRTUAL CLUSTER` only supports inspecting virtual cluster status as part of the [physical cluster replication]({% link {{ page.version.version }}/physical-cluster-replication-overview.md %}) workflow.
```

Comment:
Is it worth adding this text to an include and using that on all the pages? I can show you how to do a parameterized include if you want it to say slightly different things on different pages.

Reply:
I'm not sure, for the same reason as a few comments above, I kind of think untangling this at a later point might be annoying.